pyo3-macros-backend/src/params.rs

```diff
@@ -100,9 +90,8 @@ pub fn impl_arg_params(
         .iter()
         .enumerate()
         .filter_map(|(i, arg)| {
-            let from_py_with = &arg.attrs.from_py_with.as_ref()?.value;
-            let from_py_with_holder =
-                syn::Ident::new(&format!("from_py_with_{}", i), Span::call_site());
+            let from_py_with = &arg.from_py_with().as_ref()?.value;
```

Comment:
I think we can drop the `.as_ref()` now.

```suggestion
            let from_py_with = &arg.from_py_with()?.value;
```